core/domain/suggestion_services.py

```diff
@@ -542,3 +556,65 @@ def check_can_resubmit_suggestion(suggestion_id, user_id):
     suggestion = get_suggestion_by_id(suggestion_id)
 
     return suggestion.author_id == user_id
+
+
+def check_can_edit_suggestion(user, suggestion_id):    # pylint: disable=too-many-return-statements
```

Comment:
I don't think this is a correct layer for this check, can we move this to acl?

Reply:
I think yes we can move into acl.
But can you tell me

1. Why should I move this to acl?
2. Why other function like ``` check_can_resubmit_suggestion( ) ```  is present in  ``` core/domain/suggestion_services.py  ```

Reply:
ping @DubeySandeep

Reply:
I'm not sure, can you please check!